src/command_line_parser.cc

```diff
@@ -1527,20 +1566,57 @@ TritonParser::ParseBackendConfigOption(const std::string& arg)
 }
 
 std::tuple<std::string, std::string, std::string>
-TritonParser::ParseHostPolicyOption(const std::string& arg)
+TritonParser::ParseGrpcRestrictedProtocolOption(const std::string& arg)
 {
-  // Format is "<policy_name>,<setting>=<value>"
-  int delim_name = arg.find(",");
-  int delim_setting = arg.find("=", delim_name + 1);
+  try {
+    return ParseGenericConfigOption(arg, ":", "=");
+  }
+  catch (const ParseException& pe) {
+    // catch and throw exception with option specific message
```

Comment:
Nit: optional. I also don't know if this comment is necessary/helpful. Your code is self-explanatory.

Reply:
I am going to keep this comment to emphasis that is re-throwing the exception for better messaging.

Reply:
"option" refers to this command line option

Reply:
Thanks for clarifying. For the future, if you hyphenate `option-specific`, that could make it clearer.